### DISCUSSION TITLE
More Github project labels

### DISCUSSION THREAD

**Discussion by LaszloGombos** [systemd/mkosi#2400] [General]
How do you feel about adding some/all of the following Github labels to the project ?

- `boot` 
- `initramfs-generator` 
- `initramfs` 
- `initramfs-image`

**Comment by daandemeyer** [comment]
Why? Can you explain the added benefit these labels would provide?

**Reply by LaszloGombos** [reply]
These labels are used to discover initramfs generators on GitHub - I assume this is also why the existing labels added to the project. 

Labels can help people/distro's to discover what mkosi project is capable of - especially now that mkosi-initrd is merged into mkosi. 

I do think all four labels are appropriate, but obviously it make sense to discuss each label one-by-one.

**Reply by daandemeyer** [reply]
Ah I misunderstood I thought this was about issue labels, not repository labels. Adding these as repository labels makes a lot of sense, I've gone ahead and done so now.
